src/CommonDBTM.php

```diff
@@ -6861,4 +6861,22 @@ public static function getByUuid(string $uuid): ?self
 
         return null;
     }
+
+    /**
+     * @param 'ASC'|'DESC' $order
+     */
+    public static function displayList(
+        array $criteria,
+        int $sort_search_option_id,
+        string $order = 'ASC'
+    ): void {
+        Search::showList(static::class, [
```

Comment:
Is that new method really needed/useful?
I'm a bit surprised to see a new CommonDBTM method used only once in this small PR :)

Seems OK appart of that.

Reply:
I find myself always needing to copy the same code and parameters when I went to display a simple search result (I can find at least 5 similar calls in the code).

I guess they could be updated to use this method once this is merged.

Reply:
I'm OK if there are other possibles usages.